VRCFaceTracking/SDK/ExtTrackingModule.cs

```diff
@@ -5,13 +5,13 @@ namespace VRCFaceTracking
     public abstract class ExtTrackingModule
     {
         // Should UnifiedLibManager try to initialize this module if it's looking for a module that supports eye or lip.
-        public virtual (bool SupportsEye, bool SupportsLip) Supported => (false, false);
-        
+        public virtual (bool SupportsEye, bool SupportsExpressions) Supported => (false, false);
```

Comment:
I made the change from SupportsLip to SupportsExpressions early, though notably I accidentally made it plural whereas all other instances of 'expression' are singular, so maybe it should be SupportsExpression? Idk it makes sense either way, but we should be consistent with it (more-so my fault).

Reply:
Resolved in https://github.com/benaclejames/VRCFaceTracking/pull/122/commits/aa938e3d5e4e96673962590d56e10897764e2fee